fili-core/src/test/java/com/yahoo/bard/webservice/data/config/dimension/TestLookupDimensionConfig.java

```diff
@@ -13,37 +18,88 @@
  * Hold all of the Configuration information for the look up dimensions.
  */
 public class TestLookupDimensionConfig extends TestDimensionConfig implements LookupDimensionConfig {
-    private List<String> namespaces;
+    private final List<String> namespaces;
```

Comment:
Since this is for testing, it's OK for it to be non-final. Tests often need to reach inside things and mess with them, and `final` makes that hard to do. For non-test code, `final` is good, because most things should be immutable, and `final` helps a lot with that.